qt/DetailsDialog.cc

```diff
@@ -849,6 +889,39 @@ void DetailsDialog::refreshUI()
 
     ui_.privacyValueLabel->setText(string);
 
+    // myLabelsTextEdit
+    string = none;
+
+    if (labels_need_refresh_)
+    {
+        if (torrents.empty())
+        {
+            labels_baseline_.clear();
+            ui_.labelsTextEdit->setText({});
+            ui_.labelsTextEdit->setPlaceholderText(none);
+            ui_.labelsTextEdit->setReadOnly(true);
+            ui_.labelsTextEdit->setEnabled(true);
```

Comment:
should this be false?

Reply:
This is here for visual consistency reasons with the comment text box bellow. Both are enabled but read-only when no torrents are selected and only get disabled when torrents are mixed.